src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java

```diff
@@ -19,6 +25,35 @@ public UtilityPCAPDaoImpl() {
 
   }
 
+  public String getSessionEntity(HttpServletRequest request) {
+    HttpSession session = request.getSession(false);
+    try{  
+      String n = (String)session.getAttribute("entityName");  
+      return n;
+       }
+      catch(Exception e)
+       {
+          return null;
+        }
+  }
+
+  public void setSessionEntity(HttpServletRequest request, String text) {
+    HttpSession session = request.getSession();  
+    session.setAttribute("entityName",text);  
+  }
```

Comment:
These methods dont belong in a Utility Class, it is a separate class for session Management, like `class SessionManager`

Reply:
nevermind keep it here for now but make sure to keep all methods static.

Reply:
ran into issues declaring static methods while having an interface, ended up created the SessionManager class